packages/scss/src/components/indexTable/component.scss

```diff
@@ -0,0 +1,192 @@
+                --components-indexTable-outline-border-radius: 0 var(--components-indexTable-outline-border-radius-default) var(--components-indexTable-outline-border-radius-default) 0;
+            }
+
+            &::before,
+            &::after {
+                content: '';
+                position: absolute;
+                z-index: -1;
+                transition-duration: var(--commons-animations-durations-fast);
+            }
+
+            // the "card" apparence is put on a ::before pseudo element
+            &::before {
+                inset: var(--components-indexTable-cell-inset-y, 0) var(--components-indexTable-cell-inset-x-right, 0) var(--components-indexTable-cell-inset-y, 0) var(--components-indexTable-cell-inset-x-left, 0);
+                border-radius: var(--components-indexTable-cell-border-radius, 0);
+                background-color: var(--components-indexTable-cell-background-color, var(--colors-white-color));
+                // shadow-mask : since the card apparence is put on every cell (du to a Safari bug), we need to hide the left part of the box shadow for everycell exept the first one.
+                // We do this with a rectangular white box-shadow above it.
+                box-shadow: var(--components-indexTable-shadow-mask), var(--components-indexTable-cell-shadow, var(--components-indexTable-cell-shadow-default));
+                transition-property: inset, box-shadow, background-color;
+            }
+
+            // the focus outline, when needed
+            &::after {
+                inset: calc(var(--components-indexTable-cell-inset-y, 0px) - var(--components-indexTable-outline-offset)) calc(var(--components-indexTable-cell-inset-x-right, 0px) - var(--components-indexTable-outline-offset))
+                    calc(var(--components-indexTable-cell-inset-y, 0px) - var(--components-indexTable-outline-offset)) calc(var(--components-indexTable-cell-inset-x-left, 0px) - var(--components-indexTable-outline-offset));
+                border-radius: var(--components-indexTable-outline-border-radius);
+                border-width: var(--components-indexTable-outline-border-width);
+                border-style: solid;
+                border-color: var(--components-indexTable-outline-color);
+                opacity: var(--components-indexTable-outline-opacity, 0);
+                transition-property: inset, opacity;
+            }
+        }
+
+        .indexTable-body-row-cell-checkbox,
+        .indexTable-head-row-cell-checkbox {
+            display: block;
+        }
+
+        .indexTable-head-row-cell-checkbox {
+            margin-top: var(--spacings-XS);
+        }
+
+        .indexTable-body-row-cell-checkbox {
+            margin-top: 0.75rem;
+        }
+
+        //hidden element that take focus when the row is focused
+        .indexTable-body-row-cell-action {
+            @include a11y.mask;
```

Comment:
I find it strange that this link is hidden by default, and I think that each BU should be allowed to do this as it sees fit.

Reply:
Since the whole row is technicaly the link, i don't see how a visible link would work with the current design. specificaly with the row hover / focus / outlined states. That would be unconsistent with others clickables elements that could be found inside the row, likes actions.